src/config/routes.ts

```diff
@@ -145,4 +145,9 @@ interface StageStatusResponseItem {
 export function isCIPipelineCreated(responseArr: StageStatusResponseItem[]): boolean {
     let ciPipeline = responseArr.find((item) => item.stageName === 'CI_PIPELINE');
     return ciPipeline.status;
+}
+
+export function isCDPipelineCreated(responseArr: StageStatusResponseItem[]): boolean {
+    let cdPipeline = responseArr.find((item) => item.stageName === 'CD_PIPELINE');
+    return cdPipeline.status;
```

Comment:
Please put check here to prevent `Cannot read....of undefined` error.

Reply:
It won't be empty. responseArr will always have a row containing 'CD_PIPELINE' stageName

Reply:
Got it. Please add the check to make it fail-safe.